src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = days * 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days >= 7) {
+    return dailyCost - bigDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
will work just fine without the second check
```suggestion
  if (days >= 3) {
```